niceday-api/controllers/UserTrackers.js

```diff
@@ -10,3 +10,14 @@ module.exports.setUserTrackerStatuses = function setUserTrackerStatuses(req, res
       utils.writeJson(res, response);
     });
 };
+
+// eslint-disable-next-line no-unused-vars
+module.exports.getSmokingTrackerData = function getSmokingTrackerData(req, res, next, body) {
+  UserTrackers.getSmokingTrackerData(req)
+    .then((response) => {
+      utils.writeJson(res, response);
+    })
+    .catch((response) => {
+      utils.writeJson(res, response);
```

Comment:
This always looks like a bit of a weird pattern, but I suppose we don't actually have any error handling code we could be running here anyway...?

Reply:
yeah, I also always find this a bit weird.. We can add logging here at some point.